pymeasure/display/manager.py

```diff
@@ -139,7 +139,9 @@ def __init__(self, plot, browser, port=5888, log_level=logging.INFO, parent=None
         self._monitor = None
         self.log_level = log_level
 
-        self.plot = plot
+        if not type(plots) == list:
```

Comment:
I think it's more common to check this with `isinstance`

Reply:
I will change that then